MykSuite/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+  ~ Infomaniak Core - Android
+  ~ Copyright (C) 2025 Infomaniak Network SA
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<resources>
+    <string name="buttonClose">Cerrar</string>
+    <string name="myKSuiteUpgradeDescription">Más espacio, más posibilidades y más flexibilidad para hacer lo que te gusta.</string>
+    <string name="myKSuiteUpgradeDetails">Para beneficiarse de mi kSuite+, modifique su oferta desde su interfaz web.</string>
```

Comment:
The description uses the informal tone and the details uses the formal tone

Reply:
Can you correct it pls ?